README.md

```diff
@@ -1 +1,23 @@
-# shop-be
\ No newline at end of file
+# shop-be
+
+Shop Info Service directory includes 2 lambda methods:
+- getProductsList(https://lf379wln0g.execute-api.eu-west-1.amazonaws.com/dev/get-products-list)
+  - GET method
+  - Receives ether 200 OK or 500 
+  
+- getProductById(https://lf379wln0g.execute-api.eu-west-1.amazonaws.com/dev/get-product-by-id?=122)
```

Comment:
id should be implemeted as path parameter

Reply:
updated